Provide a fallback implementation of sincos for OS X 10.8 and earlier
which don't have __sincos.
https://github.com/gnudatalanguage/gdl/commit/a7dbf138ba4fda9e428ceab15b9ea7b7f7bc0948
--- src/ssrfpack.c.orig	2018-12-01 08:19:52.000000000 -0600
+++ src/ssrfpack.c	2019-10-04 06:30:06.000000000 -0500
@@ -3,8 +3,12 @@
  * are not required to compile and link the sph supplement.
  */
 #ifdef __APPLE__
+#if MAC_OS_X_VERSION_MIN_REQUIRED >= 1090
 #define sincos(x, s, c) __sincos(x, s, c)
 #define sincosf(x, s, c) __sincosf(x, s, c)
+#else
+#define sincos(x,s,c) (*s = sin(x), *c = cos(x))
+#endif
 #endif
 
 #ifdef __FreeBSD__
